src/main/java/com/rog/EShop/properties/KeycloakProperties.java

```diff
@@ -0,0 +1,37 @@
+package com.rog.EShop.properties;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConfigurationProperties("keycloak")
+
+public class KeycloakProperties {
+    private String host;
+    private String clientId;
+    private String clientSecret;
+
```

Comment:
add also `String realm` to `KeycloakProperties`, and change everywhere url `"/realms/ESTORE/..."` to 
```
"/realms/" + keycloakProperties.getRealm() + "/..."
```